base/operators.jl

```diff
@@ -997,7 +997,7 @@ Represents the composition of two callable objects `outer::Outer` and `inner::In
 ```julia
 ComposedFunction(outer, inner)(args...; kw...) === outer(inner(args...; kw...))
 ```
-The preferred way to construct instance of `ComposedFunction` is to use the composition operator [`∘`](@ref):
+The preferred way to construct an instance of `ComposedFunction` is to use the composition operator [`∘`](@ref):
```

Comment:
Unrelated docfix?

Reply:
Oops indeed. Should I make another PR for this?

Reply:
Probably good, yes - small doc fixes can be merged much faster. You can force-push a new version without that change to the branch you make this PR from to update this PR.

Reply:
OK!